src/pyhf/infer/calculators.py

```diff
@@ -173,13 +194,24 @@ def __init__(
             pdf (~pyhf.pdf.Model): The statistical model adhering to the schema ``model.json``.
             init_pars (:obj:`tensor`): The initial parameter values to be used for fitting.
             par_bounds (:obj:`tensor`): The parameter value bounds to be used for fitting.
-            fixed_params (:obj:`tensor`): Whether to fix the parameter to the init_pars value during minimization
-            test_stat (:obj:`str`): The test statistic to use as a numerical summary of the data.
-            qtilde (:obj:`bool`): When ``True`` perform the calculation using the alternative
-             test statistic, :math:`\tilde{q}_{\mu}`, as defined under the Wald
-             approximation in Equation (62) of :xref:`arXiv:1007.1727`
-             (:func:`~pyhf.infer.test_statistics.qmu_tilde`).
-             When ``False`` use :func:`~pyhf.infer.test_statistics.qmu`.
+            fixed_params (:obj:`tensor`): Whether to fix the parameter to the init_pars value
+              during minimization.
+            test_stat (:obj:`str`): The test statistic to use as a numerical summary of the
+              data: ``'qtilde'``, ``'q'``, or ``'q0'``.
+              ``'qtilde'`` (default) performs the calculation using the alternative test statistic,
+              :math:`\tilde{q}_{\mu}`, as defined under the Wald approximation in Equation (62)
+              of :xref:`arXiv:1007.1727` (:func:`~pyhf.infer.test_statistics.qmu_tilde`), ``'q'``
+              performs the calculation using the test statistic :math:`q_{\mu}`
+              (:func:`~pyhf.infer.test_statistics.qmu`), and ``'q0'`` perfoms the calculation using
+              the discovery test statistic :math:`q_{0}` (:func:`~pyhf.infer.test_statistics.q0`).
+            calc_base_dist (:obj:`str`): The statistical distribution, ``'normal'`` or
+              ``'clipped_normal'`` to use for calculating the :math:`p`-values.
+              ``'normal'`` (default) corresponds to the common misconception that :math:`p`-values
+              greater than the observed :math:`p`-value are obtainable for POI less than 1.
+              ``'clipped_normal'`` corresponds to limiting the :math:`p`-values for expected limits
+              to a maximum of the observed :math:`p`-value.
+              The choice of ``calc_base_dist`` only affects the :math:`p`-values for expected limits,
+              and the default value will be changed in a future release.
```

Comment:
@lukasheinrich Please review this (c.f. the linked render on RTD in the PR body) for accuracy. I tried to keep this concise, but it is more important to be clear and correct.

Reply:
i'd say


 `normal: use full normal distribution in µ/sigma space. note: expected limits may correspond to scenarios   with  $expected µ^ > µ$`

 `clipped_normal: use clipped normal distribution in µ/sigma space. to avoid expected limits that correspond to scenarios   with  $expected µ^ > µ$`

Reply:
Updated. :+1: